docs/integrations/sources/stripe.md

```diff
@@ -1,2 +1,46 @@
 # Stripe
 
+## Overview
+
+The Stripe source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
+
+This Stripe source is based on the [Singer Stripe Tap](https://github.com/singer-io/tap-stripe).
+
+### Output schema
+
+Several output streams are available from this source \(customers, charges, invoices, subscriptions, etc.\) For a comprehensive output schema [look at the Singer tap schema files](https://github.com/singer-io/tap-stripe/tree/master/tap_stripe/schemas).
+
+### Data type mapping
```

Comment:
It seems like this tap already flattens nested data (tables), and stripe doesn't output funky data types, so there isn't any "interesting" type mapping going on. So can we mention that all of Stripe's types are first class types in JsonSchema and therefore there is no "mapping" going on? Might just be easer to have a table that goes

string -> string
date -> date

etc...

Reply:
clarified